doc/client_lua_api.txt

```diff
@@ -1388,12 +1388,23 @@ Displays a horizontal bar made up of half-images.
 * `offset`: offset in pixels from position.
 
 ### `waypoint`
-Displays distance to selected world position.
+
+Displays distance to selected world position. Accepts offset.
 
 * `name`: The name of the waypoint.
 * `text`: Distance suffix. Can be blank.
-* `number:` An integer containing the RGB value of the color used to draw the text.
+* `precision`: Waypoint precision. Defaults to 10 if not set (for compatibility reasons).
```

Comment:
Does this support fractional values? Document please

Reply:
No, it does not (because the used item field is an uint)